### DISCUSSION TITLE
The issue is closed as a duplicate, but nobody is contributing

### DISCUSSION THREAD

**Discussion by dushmanta05** [simple-icons/simple-icons#8684] [General]
Hello there! I wanted to contribute an icon, so I opened an issue and also created a PR, and it passed all the checks, but someone else had created an issue for that icon before 2 years ago, and that's why my issue got closed for duplicate. Also in that issue, he has opted *no* in the **contributing** field, so I guess he isn't going to contribute since it's been 2 years. So I want to ask, "Can I contribute this icon and be assigned it to that issue?

**Comment by jwr1** [comment]
You can can pretty much contribute to whatever icon you want, it doesn't mater if you opened the issue or someone else, as long as someone isn't already working on it, which in this case, you are (so you're good :+1:). The reason it's not being merged is because no one with merge permissions has checked it out yet, so you just have to be patient and wait. I would recommend fixing the size issue I pointed out eariler, make sure the automated tests pass again, and then wait for a maintainer to review it (which can take between a few days, to a month or two).

**Reply by dushmanta05** [reply]
Thanks for your response @jwr12135 . Yes, I will finish the size issue you've mentioned in my PR. Although I'm asking because my issue #8640 got closed as a duplicate of #6950, I wanted to know if I would still be able to contribute or not. Thanks for answering that. Also in my PR #8593, I mentioned my issue #8640, which would have been closed after a successful merge. Now that it's closed, can I mention the first issue, #6950, there? So that it would indicate I'm working on that issue.

**Reply by jwr1** [reply]
> I mentioned my issue https://github.com/simple-icons/simple-icons/issues/8640, which would have been closed after a successful merge. Now that it's closed, can I mention the first issue, https://github.com/simple-icons/simple-icons/issues/6950, there?

Yes, it helps the author of the issue (and other people) know you're working on it; additionally, it will automatically close the issue when the pull request is closed.

**Reply by dushmanta05** [reply]
Okay, thanks for conforming.
